android/src/app/organicmaps/editor/EditorHostFragment.java

```diff
@@ -309,7 +309,7 @@ public void onClick(View v)
         else
         {
           // TODO (yunikkk) correct translation
```

Comment:
Why the todo is not fixed? Why the existing string is not updated in strings.txt (see docs/TRANSLATIONS.md) and a new string is created instead?